### DISCUSSION TITLE
stocks and portfolio management

### DISCUSSION THREAD

**Discussion by Resurrectum** [ledgersmb/LedgerSMB#7348] [Design]
Can SMB Ledger manage stocks and portfolios?
Is there an automated lookup of exchange rates?

**Comment by ehuelsmann** [comment]
> Can SMB Ledger manage stocks and portfolios? 

You could use invoices to buy and sell stocks. The inventory tracking module will then track the number of each bond/share that have been bought/sold. You will need to make manual adjustments to account for the (daily) change in the value of the traded items.

> Is there an automated lookup of exchange rates?

Yes. However, the connection that we have is linked to a source that wants to be paid for commercial use. This is a third-party that isn't related to our project, but we couldn't find any party that would be providing rates for free to everybody. (Meaning to say: the project doesn't earn anything from the connection.)

**Reply by promexio** [reply]
Is the mentionned interface retrieving currency exchange rates or stock exchange? I guess the initial question was dedicated to a financial asset managment. 
I'd also be very curious if such a "module" is available...

**Reply by ehuelsmann** [reply]
Currency exchange, is what is meant.
